### PR TITLE
Add list item component

### PR DESCRIPTION
**Pull request by radeknovis**

### CODE REVIEW

File: Sources/Web3Modal/Sheets/Web3ModalView.swift [hunk 1]
```diff
@@ -40,7 +40,7 @@ struct Web3ModalView: View {
                 EmptyView()
             case .getWallet:
                 GetAWalletView(
-                    wallets: Wallet.stubList
+                    wallets: []
```
Comment (flypaper0) [inline Sources/Web3Modal/Sheets/Web3ModalView.swift]: why stubs replaced with empty array?

File: Sources/Web3ModalUI/Miscellaneous/CircleProgressView.swift [hunk 2]
```diff
@@ -0,0 +1,268 @@
...
+//        path.move(to: CGPoint(x: rect.midX, y: rect.minY))
+//        path.addLine(to: CGPoint(x: rect.maxX - cornerRadius, y: rect.minY))
+//        path.addArc(tangent1End: CGPoint(x: rect.maxX, y: rect.minY), tangent2End: CGPoint(x: rect.maxX, y: rect.minY + cornerRadius), radius: cornerRadius)
+//        path.addLine(to: CGPoint(x: rect.maxX, y: rect.maxY - cornerRadius))
+//        path.addArc(tangent1End: CGPoint(x: rect.maxX, y: rect.maxY), tangent2End: CGPoint(x: rect.maxX - cornerRadius, y: rect.maxY), radius: cornerRadius)
+//        path.addLine(to: CGPoint(x: rect.minX + cornerRadius, y: rect.maxY))
+//        path.addArc(tangent1End: CGPoint(x: rect.minX, y: rect.maxY), tangent2End: CGPoint(x: rect.minX, y: rect.maxY - cornerRadius), radius: cornerRadius)
+//        path.addLine(to: CGPoint(x: rect.minX, y: rect.minY + cornerRadius))
+//        path.addArc(tangent1End: CGPoint(x: rect.minX, y: rect.minY), tangent2End: CGPoint(x: rect.minX + cornerRadius, y: rect.minY), radius: cornerRadius)
+//        path.closeSubpath()
```
Comment (flypaper0) [inline Sources/Web3ModalUI/Miscellaneous/CircleProgressView.swift]: may be add todo or remove comments?

**Review by flypaper0 — APPROVED** [review]

File: Sources/Web3ModalUI/Miscellaneous/CircleProgressView.swift (hunk 2 above)
Comment (radeknovis) [reply Sources/Web3ModalUI/Miscellaneous/CircleProgressView.swift]: Removed, good spot

File: Sources/Web3Modal/Sheets/Web3ModalView.swift (hunk 1 above)
Comment (radeknovis) [reply Sources/Web3Modal/Sheets/Web3ModalView.swift]: Reverted

**Pull request merged**